Another projekts/calculator/main_calculator_1.py

```diff
@@ -0,0 +1,37 @@
+run = True
+
+while run:
+    what_todo = input("What do you want to do?\n1 for +\n2 for -\n3 for *\n4 for /\n: ")
+
+    def mathematical_operation():
```

Comment:
This is an independent function. Is there any Python standard that requires function creation in the while loop?

Reply:
Please do.

Reply:
Done